parsers/UA.py

```diff
@@ -38,24 +39,27 @@
 
 def fetch_production(
     zone_key: str = "UA",
-    session: Optional[Session] = None,
     target_datetime: Optional[datetime] = None,
     logger: Logger = getLogger(__name__),
 ) -> list:
     if target_datetime:
-        raise NotImplementedError("This parser is not yet able to parse past dates")
-    r = session or Session()
+        target_date = arrow.get(target_datetime.date()).format("DD.MM.YYYY")
+    else:
+        target_date = arrow.now().format("DD.MM.YYYY")
```

Comment:
The old version injected the timezone here as well, something I think we should be doing in the new version as well.